helpers/github.ts

```diff
@@ -62,6 +62,22 @@ export async function forceCloseMissingIssues(devpoolIssues: GitHubIssue[], proj
   }
 }
 
+/**
+ * Stops forks from spamming real Ubiquity issues with links to their forks
+ * @returns true if the authenticated user is Ubiquity
+ */
+export async function checkIfForked() {
+  try {
+    const {
+      data: { id },
+    } = await octokit.users.getAuthenticated();
```

Comment:
I'm a little bit concerned that it will authenticate as the bot itself instead of the organization. But I asked ChatGPT and it said that this implementation should be fine.

https://chat.openai.com/share/47cc89cc-cd67-4f68-931f-fe6e1b81ba5e